installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -167,6 +228,13 @@ spec:
         - name: {{ kubernetes_deployment_name }}-web
           image: "{{ kubernetes_web_image }}:{{ kubernetes_web_version }}"
           imagePullPolicy: Always
+{% if web_security_context is defined and web_security_context == 'true' %}
```

Comment:
This is where the mixed typing is funny and might lead to odd behavior.  I'd rather use something like the jinja/ansible type casting to pick up other boolean variation `|bool` for example.

Reply:
resolved inline with the changes in conversion 1